woocommerce/woocommerce-admin
# Add track data for the preview site btn

Fixes #6580 

This PR adds tracking data when a user clicks the `Preview site` button from the activity panel.

### Detailed test instructions:

1. Navigate to WooCommerce -> Home and click "Personalize your store" from the OBW task list or navigate to `/wp-admin/admin.php?page=wc-admin&task=appearance`
2. Open your inspector and enable debugging mode by executing `localStorage.setItem( 'debug', 'wc-admin:*' );` 
3. Click the `Preview site` button and confirm it contains the following event data.

Prop | Possible Values | Note
-----|-----------------|-------
create_homepage| Y or N | when user created homepage from step #2
upload_logo| Y or N | when user uploads a logo from step #3
set_notice | Y or N | when user sets notice text from step #4
